cohere/client.py

```diff
@@ -199,6 +200,25 @@ def classify(
 
         return Classifications(classifications)
 
+    def moderate(
+        self,
+        inputs: List[str],
+        model: str = None
+    ) -> Moderations:
+        json_body = json.dumps({
+            'model': model,
+            'inputs': inputs,
+        })
+        response = self.__request(json_body, cohere.MODERATE_URL)
+
+        moderations = []
+        for res in response['results']:
+            moderations.append(
+                Moderation(res['benign'], res['profanity'], res['hate_speech'], res['violence'], res['self_harm'],
```

Comment:
are we sticking with `profanity` rather than `profane`?

Reply:
we can stick with this for now since it's live